client/my-sites/plugins/plugins-browser-list/index.jsx

```diff
@@ -106,6 +110,15 @@ const PluginsBrowserList = ( {
 					) }
 				</div>
 			</div>
+			{ listName === 'paid' && isEnabled( 'marketplace-spotlight' ) && spotlightPluginFetched && (
```

Comment:
The spotlight component loads fine with my a11n user but wsods with my test user: 

![Screenshot_2022-02-21_12-15-42](https://user-images.githubusercontent.com/811776/154873927-cedca86f-7afe-4c36-ad93-560a9ad0bb8d.png)

I seem to recall @cpapazoglou mentioning this `wordpress-seo-premium` plugin is only available to a11n users? 

This suggestion will prevent the wsod but the 404 still shows in console after applying so there is probably more to do.

```suggestion
			{ listName === 'paid' && isEnabled( 'marketplace-spotlight' ) && spotlightPluginFetched &&  spotlightPlugin && (
```

Reply:
good catch!

Reply:
Yeah the 404 will still happen because the hook is still being called, I don't think we can conditionally fetch without needing to change the hook API but this seems like a good follow-up!

Reply:
Added it as a follow-up issue here: https://github.com/Automattic/wp-calypso/issues/61323